src/js/actions/guides.js

```diff
@@ -114,9 +114,13 @@ define(function (require, exports) {
             topAncestors = currentDocument.layers.selectedTopAncestors,
             topAncestorIDs = collection.pluck(topAncestors, "id"),
             visibleGuides = guides.filter(function (guide) {
-                return guide.layerID === 0 || topAncestorIDs.has(guide.layerID);
+                return guide && (guide.layerID === 0 || topAncestorIDs.has(guide.layerID));
```

Comment:
Guides are stored by index, and we listen to `set` events from Photoshop to change our guide structure.

If the user is clicking fast enough we can get into a situation where we get the guide index n+1 before we get guide index n's set event. In these cases, because we store guides in an indexed list, index n will be undefined until that event is processed, but guide policies may need updating between those two moments, necessitating the null check.